packages/oats/src/generate-types.ts

```diff
@@ -1497,7 +1497,7 @@ export function run(options: Options) {
   }
 
   function resolveRefToTypeName(ref: string): { qualified?: ts.Identifier; member: string } {
-    const external = options.resolve(ref, options);
+    const external = options.resolve(ref, options, 'value'); //TODO: value of 'kind' in the context is unsure
```

Comment:
How do we determine kind of ref in this context, @sakari ?

Reply:
I'd say that you have to pass it from the caller of `resolveRefToTypeName`